wp-content/themes/core/assets/js/src/utils/dom/accessibility.js

```diff
@@ -0,0 +1,50 @@
+import * as tools from 'utils/tools';
+
+/**
+ * @function focusLoop
+ * @description Loop through focusable els inside a container. Bound to a keydown listener usually.
+ *
+ * @param {KeyboardEvent} e
+ * @param {HTMLElement} trigger
+ * @param {HTMLElement} container
+ * @param {Function} onEscape
+ */
+
+const focusLoop = ( e = {}, trigger = null, container = null, onEscape = () => {} ) => {
```

Comment:
🙌  this is great! super handy

Reply:
All props to Sam for this method. I can't take any credit. :)